src/rustdoc/meetings.md

```diff
@@ -0,0 +1,16 @@
+# Meetings
+
+The rustdoc team hosts a meeting every second Monday on each month on the 
+[t-rustdoc/meetings channel on zulip](https://rust-lang.zulipchat.com/#narrow/channel/393423-t-rustdoc.2Fmeetings)
+at 21:00 CET (UTC+1) and CEST on summer (UTC+2).
```

Comment:
Seems better to just link to the calendar here since this will probably fall out of date.

Reply:
The link is just below.